.github/workflows/ci.yaml

```diff
@@ -0,0 +1,34 @@
+name: CI
+on: [push, pull_request, workflow_dispatch]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v2
+
+    - name: Setup Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.15.3
+     
+    - name: CI - Verifications and Tests
+      run: |
+        make toolchain 
+        echo "/usr/local/kubebuilder/bin" >> $GITHUB_PATH
+        make ci
+
+    - name: Upload code coverage
+      uses: actions/upload-artifact@v2
+      with:
+        name: coverage
+        path: coverage.html
+
+    - name: Verify Licenses
+      if: github.event_name == 'push'
```

Comment:
Should we just always check licenses?

Reply:
we can't, github rate limits pretty aggressively so we need GITHUB_TOKEN which is only available on push

Reply:
ha we can, get call